doc/howtos/backend.rst

```diff
@@ -115,6 +115,20 @@ or XML. The usage of most of those files will be explained along this tutorial.
 
         .. patch::
 
+Odoo does not display my module in the Apps list
+------------------------------------------------
+
+Odoo **does not** refresh the Apps list when starting the server. This may
+lead to unintuitive behavior: you write (the skeleton of) a new module,
+you add the *module directory* to the ``--addons-path`` and... you don't find
+it once the server is restarted.
+
+In order to update the list of available modules, switch to the *Developer
+mode*, then go to :menuselection:`Apps --> Update Apps List --> Update`.
+
+If you still don't find your module in the Apps list, this may be due to an
+incorrect module structure or a bad ``--addons-path``.
+
```

Comment:
tip: when you install or update a module through `-i` or `-u`, the list is automatically refreshed ;)

Reply:
Even before 11.0 ?

For the records: at this step of the tutorial, `-i` and `-u` options are not explained yet